domain/cloud/state/state.go

```diff
@@ -141,28 +141,29 @@ func (st *State) Cloud(ctx context.Context, name string) (*cloud.Cloud, error) {
 	return result, errors.Trace(err)
 }
 
-// GetCloudForID returns the cloud associated with the provided id. If no cloud is
-// found for the given id then a [clouderrors.NotFound] error is returned.
-func (st *State) GetCloudForID(ctx context.Context, id corecloud.ID) (cloud.Cloud, error) {
+// GetCloudForUUID returns the cloud associated with the provided uuid. If no
+// cloud is found for the given id then a [clouderrors.NotFound] error is
+// returned.
+func (st *State) GetCloudForUUID(ctx context.Context, id corecloud.UUID) (cloud.Cloud, error) {
 	db, err := st.DB()
 	if err != nil {
 		return cloud.Cloud{}, errors.Trace(err)
 	}
 
 	var rval cloud.Cloud
 	return rval, db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
-		rval, err = GetCloudForID(ctx, st, tx, id)
+		rval, err = GetCloudForUUID(ctx, st, tx, id)
 		return err
 	})
 }
 
-// GetCloudForID returns the cloud associated with the provided id. If no cloud is
+// GetCloudForUUID returns the cloud associated with the provided id. If no cloud is
 // found for the given id then a [clouderrors.NotFound] error is returned.
-func GetCloudForID(
+func GetCloudForUUID(
 	ctx context.Context,
 	st domain.Preparer,
 	tx *sqlair.TX,
-	id corecloud.ID,
+	id corecloud.UUID,
```

Comment:
id -> uuid

Reply:
I agree with @jack-w-shaw here.

```suggestion
	uuid corecloud.UUID,
```

Reply:
Ah yea, I missed that one by accident.